salt/_modules/metalk8s_package_manager_yum.py

```diff
@@ -37,9 +37,6 @@ def _list_dependents(
         '{}-{}'.format(name, version)
     ]
 
-    if fromrepo:
-        command.extend(['--disablerepo', '*', '--enablerepo', fromrepo])
-
```

Comment:
Oh yes we have an if my bad. Then for me we should keep this to have a module able to handle `fromrepo` even if we not use it